go.mod

```diff
@@ -9,15 +9,18 @@ require (
 	github.com/klauspost/compress v1.17.6
 	github.com/minio/md5-simd v1.1.2
 	github.com/rs/xid v1.5.0
-	golang.org/x/crypto v0.21.0
-	golang.org/x/net v0.23.0
+	golang.org/x/crypto v0.25.0
+	golang.org/x/net v0.27.0
 	gopkg.in/ini.v1 v1.67.0
 )
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/klauspost/cpuid/v2 v2.2.6 // indirect
 	github.com/stretchr/testify v1.3.0 // indirect
-	golang.org/x/sys v0.18.0 // indirect
-	golang.org/x/text v0.14.0 // indirect
+	golang.org/x/mod v0.19.0 // indirect
+	golang.org/x/sync v0.7.0 // indirect
+	golang.org/x/sys v0.22.0 // indirect
+	golang.org/x/text v0.16.0 // indirect
+	golang.org/x/tools v0.23.0 // indirect
 )
```

Comment:
do not update `go mod` if you don't intend to, please revert these changes.

Reply:
I reverted those changes sorry for the overhead.